pkg/userdata/flatcar/provider.go

```diff
@@ -374,6 +376,13 @@ storage:
         inline: |
 {{ .NodeIPScript | indent 10 }}
 
+    - path: "/etc/systemd/network/zz-default.network.d/ipv6-fix.conf"
```

Comment:
Please add the link of the flatcar issue https://github.com/flatcar-linux/Flatcar/issues/384 for context. Also please add a one-liner here regarding why this is required.